pages/docs/workflows.md

```diff
@@ -527,3 +527,40 @@ run = 'deno --unstable run -A --no-check --import-map importmap.json deno-genera
 ```
 
 For an easier `chomp deno-generate` execution that will first ensure the import map is up to date in the task graph.
+
+## Vitejs
+
+ES modules are supported directly in all major [browsers](https://caniuse.com/es6-module) now and with [es-modu-shims](https://github.com/guybedford/es-module-shims) you can polyfill the rest of the missing features and ship ES modules with confidence.
+[Vite](https://vitejs.dev/) is built heavily on the ES modules which makes using both JSPM and Vite a great combination.
+
+The dependencies are served from JSPM, and the project is bundled using Vite. We released `vite-plugin-jspm` to enable this workflow. You can download the plugin using:
+
+```sh
+npm install vite-plugin-jspm --save-dev
+```
+
+`vite` and `vite build` can be used dev-server and production builds respectively. For the dev build, JSPM will inject ES Module Shims to polyfill import maps, while in the production build all modules will be build from the CDN. The plugin takes all the options that are supported with [@jspm/generator](https://github.com/jspm/generator#options).
+For the production build, it can be useful to disable the `polyfilling` and `preload` options:
+
+vite.config.mjs
+
+```js
+import { defineConfig } from "vite";
+import jspmPlugin from "vite-plugin-jspm";
+
+export default defineConfig({
+  clearScreen: true,
+  build: {
+    polyfillModulePreload: false,
+    polyfillDynamicImport: false,
+  }
+  plugins: [jspmPlugin()],
+});
+```
+An additional option that you can use is `downloadDeps`, which helps in downloading all the dependencies at build time.
+In considering the `downloadDeps` option, there are a number of factors that should be considered:
+- Loading modules from the JSPM CDN will results in a shared edge cache with other JSPM users resulting in lower latencies for code delivery over the globally distributed CDN network.
```

Comment:
Remove duplicate "result" term. Also we should be clearer that this is a benefit:

```suggestion
- Using external CDN dependencies will likely provide the best latency by utilizing the shared globally distributed CDN network - since the JSPM CDN cache is shared at the edge with other JSPM users this leads to shared latency optimization on edges.
```